hoodie-client/src/main/java/com/uber/hoodie/io/HoodieCreateHandle.java

```diff
@@ -164,8 +159,9 @@ public WriteStatus close() {
       stat.setNumInserts(insertRecordsWritten);
       stat.setPrevCommit(HoodieWriteStat.NULL_COMMIT);
       stat.setFileId(writeStatus.getFileId());
-      stat.setPaths(new Path(config.getBasePath()), path, tempPath);
-      stat.setTotalWriteBytes(FSUtils.getFileSize(fs, getStorageWriterPath()));
+      stat.setPath(new Path(config.getBasePath()), path);
+      long fileSizeInBytes = FSUtils.getFileSize(fs, path);
```

Comment:
why separate this into a new line ? We are not using this value anywhere else anyways..

Reply:
The other PR  (Timeline-Server) uses this to set another attribute.